### PR TITLE
fix: call for speaker page messages

### LINKED ISSUE #5408: Sessions/Speakers: Call for Speakers - Public Page vs. User Page
This issue has a low priority 

1. Describe the problem: this is not a problem, it is rather a question

* Public page before user log-in, submitted session always shows No Session which is a bit confusing for the public
Screenshot from 2020-10-28 22-58-58

* User Log-In - this page is nice for users as they can see that they already submitted something 
Screenshot from 2020-10-28 22-59-45

2. Solution
Can submitted session only show when a user login

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5408 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/public/call-for-speakers.hbs [hunk 1]
```diff
@@ -38,7 +38,11 @@
   {{/unless}}
 <div class="ui hidden divider"></div>
 {{#if this.isNewSession}}
-  <p>{{t 'No session have been submitted'}}</p>
+  {{#if this.authManager.currentUser.isVerified}}
```
Comment (iamareebjamal) [inline app/templates/components/public/call-for-speakers.hbs:41]: If user is verified, and they see the message `Login to show your submitted sessions`, it'd be confusing

File: app/templates/components/public/call-for-speakers.hbs [hunk 2]
```diff
@@ -38,7 +38,11 @@
...
+    <p> {{t 'No sessions have been submitted'}} </p>
+  {{else}}
+    <p>{{t 'Login to show your submitted sessions'}}</p>
```
Comment (iamareebjamal) [inline app/templates/components/public/call-for-speakers.hbs:44]:
```suggestion
    <p>{{t 'Login to view your submitted sessions'}}</p>
```

**Comment by iamareebjamal** [comment]
@maze-runnar Status?

**Comment by maze-runnar** [comment]
What's remaining in the PR?

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-frontend/pull/5415#discussion_r514218516

File: app/templates/components/public/call-for-speakers.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/public/call-for-speakers.hbs:41]: if user is verified, they would see **No sessions have been submitted**

Comment (iamareebjamal) [reply app/templates/components/public/call-for-speakers.hbs:41]: Sorry, I meant if user is unverified

**Comment by mariobehling** [comment]
Ok, it should work like that:
* [ ] Anonymous users show message: "You need to login to submit and view sessions."
* [ ] Logged in user without sessions: "You have not submitted any sessions."
* [ ] Logged in user with sessions: Show session names.

**Comment by iamareebjamal** [comment]
@maze-runnar Please fix this soon according to the above message

File: app/templates/components/public/call-for-speakers.hbs [hunk 3]
```diff
@@ -38,14 +38,10 @@
   {{/unless}}
 <div class="ui hidden divider"></div>
 {{#if this.isNewSession}}
-  {{#if this.authManager.currentUser.isVerified}}
-    <p> {{t 'No sessions have been submitted'}} </p>
+  {{#if this.authManager.currentUser.email}}
```
Comment (iamareebjamal) [inline app/templates/components/public/call-for-speakers.hbs:41]: Wouldn't just `this.authManager.currentUser` work?

**Comment by iamareebjamal** [comment]
@mariobehling Changing `No sessions have been submitted` to `You have not submitted any sessions.` will need change in translations. Is it required?

**Comment by mariobehling** [comment]
> @mariobehling Changing `No sessions have been submitted` to `You have not submitted any sessions.` will need change in translations. Is it required?

It is always good to keep the translations up to date, yes.

**Comment by iamareebjamal** [comment]
I meant is the change required from `No sessions have been submitted` to `You have not submitted any sessions` since that'll mean previous translations becoming invalid, but I assume it is OK

File: app/templates/components/public/call-for-speakers.hbs [hunk 4]
```diff
@@ -38,7 +38,11 @@
   {{/unless}}
 <div class="ui hidden divider"></div>
 {{#if this.isNewSession}}
-  <p>{{t 'No session have been submitted'}}</p>
+  {{#if this.authManager.currentUser.email}}
```
Comment (iamareebjamal) [inline app/templates/components/public/call-for-speakers.hbs:41]:
```suggestion
  {{#if this.authManager.currentUser}}
```

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**